docker-compose.yml

```diff
@@ -162,6 +162,9 @@ services:
       BACKUP_SCHEDULE: ${BACKUP_SCHEDULE}
       KEEP_DAYS: ${KEEP_DAYS:-30}
       BACKUP_CRON_ENABLE: ${BACKUP_CRON_ENABLE:-true}
+      GF_API_KEY: ${GF_API_KEY}
+      GF_PUBLIC_HOST_AND_PORT: ${GF_PUBLIC_HOST_AND_PORT}
+      WIKIBASE_SCHEME: ${WIKIBASE_SCHEME:-https}
```

Comment:
Looks ok, shouldn't wikibase-scheme also be in the template ?

Reply:
Well, wikibase-scheme is not a new variable I have defined now, it was already introduced for the WDQS containers with fallback value 'https', as in here.

Right now it is not defined in the template, but it is also not needed in the .env file in production, because it anyways assumes the value 'https' by default. We declare it additionally as 'http' in the .env only for development.

Perhaps a better practice would be to declare it in the template as 'https' and then remove the default 'https' everytime it is called in docker-compose.yml. But I would leave that as a separate issue.